review/module.py

```diff
@@ -0,0 +1,901 @@
+                    abbreviation=abbreviation
+                )
+            )
+            await guild_log.info(
+                ctx.author,
+                ctx.channel,
+                f"Review #{result.idx} for {abbreviation} added.",
+            )
+
+    @check.acl2(check.ACLevel.MEMBER)
+    @review_subject_.command(name="add-anonymous", aliases=["anonymous", "anon"])
+    async def review_subject_add_anonymous(
+        self, ctx, abbreviation: str, grade: int, *, text: str
+    ):
+        """Add or edit anonymous subject review. If review was lastly
+        edited before 1 month, resets relevance!
+
+        Args:
+            abbreviation: Subject's abbreviation
+            grade: 1-5 (one being best)
+            text: Text of your review
+        """
+        abbreviation = abbreviation.upper()
+
+        result = await self._add_subject_review(
+            ctx, abbreviation, grade, text, anonymous=True
+        )
+
+        if result is not None:
+            await ctx.send(
+                _(ctx, "Your review of subject {abbreviation} added.").format(
+                    abbreviation=abbreviation
+                )
+            )
+            await guild_log.info(
+                ctx.author,
+                ctx.channel,
+                f"Review #{result.idx} for {abbreviation} added.",
+            )
+
+    @check.acl2(check.ACLevel.MEMBER)
+    @review_subject_.command(name="remove")
+    async def review_subject_remove(self, ctx, abbreviation: str):
+        """Remove your subject review
+
+        Args:
+            abbreviation: Subject abbreviation
+        """
+        abbreviation = abbreviation.upper()
+        subject = Subject.get_by_abbreviation(ctx, abbreviation)
+
+        if not subject:
+            await ctx.reply(
+                _(ctx, "Subject with abbreviation {abbreviation} not found.").format(
+                    abbreviation=abbreviation
+                )
+            )
+            return
+
+        review = SubjectReview.get_member_review(ctx.author, subject)
+
+        if review is None:
+            return await ctx.send(
+                _(ctx, "You have no review of subject {abbreviation}.").format(
+                    abbreviation=abbreviation
+                )
+            )
+
+        embed = self._get_subject_review_embed(
+            ctx=ctx,
+            subject=review.subject,
+            review=review,
+            delete=True,
+        )
+
+        view = ConfirmView(ctx, embed)
+        value = await view.send()
+
+        if value is None:
+            await ctx.send(_(ctx, "Deleting timed out."))
+        elif value:
+            review.delete()
+            await guild_log.info(
+                ctx.author, ctx.channel, f"Removed review for {subject.abbreviation}."
+            )
+            return await ctx.reply(_(ctx, "Your review was removed."))
+        else:
+            await ctx.send(_(ctx, "Deleting aborted."))
+
+    @check.acl2(check.ACLevel.MEMBER)
```

Comment:
The acl level check needs to change. Giving permissions to **all** members to override the reviews of other members shouldn't be allowed. 

I would change this to `ACLevel.SUBMOD`. 

same for the `review_subject_sudo_remove` command.

Reply:
I see.